cert-manager-multi-arch.md

```diff
@@ -0,0 +1,718 @@
+  "operatorframework.io/arch.amd64": "supported",
+  "operatorframework.io/arch.arm64": "supported",
+  "operatorframework.io/arch.ppc64le": "supported",
+  "operatorframework.io/arch.s390x": "supported",
+  "operatorframework.io/os.linux": "supported",
+  "provider": "Red Hat",
+  "provider-url": ""
+}
+
+```
+
+Take a look at the defaultChannel.
+```
+oc get packagemanifest openshift-cert-manager-operator -n openshift-marketplace -o json | jq .status.defaultChannel
+```
+
+```json
+"stable-v1"
+```
+
+Make a note of these informations as we'll use them when creating the `Subscription`.
+
+### Operator Installation with CLI
+
+1. Create a new project `cert-manager-operator`. This will be the operator namespace.
+
+```shell
+oc new-project cert-manager-operator
+```
+
+2. Next, we'll create the `OperatorGroup` to help OLM specify the target namespaces where the operator should be deployed and watch for it's resources.
+
+
+```shell
+oc create -f - <<EOF 
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces:
+  - "cert-manager-operator"
+EOF
+```
+
+3. Finally, create a `Subscription` to install your operator. Ensure that the information in the `spec` is sourced from the `PackageManifest` as needed.
+ 
+```shell
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  channel: stable-v1
+  name: openshift-cert-manager-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+  startingCSV: cert-manager-operator.v1.13.0
+EOF
+```
+
+
+
+4. You may do a quick verification by following the commands and sample outputs:
+
+```shell
+oc get subscription -n cert-manager-operator
+NAME                              PACKAGE                           SOURCE             CHANNEL
+openshift-cert-manager-operator   openshift-cert-manager-operator   redhat-operators   stable-v1
+
+
+oc get csv -n cert-manager-operator
+NAME                            DISPLAY                                       VERSION   REPLACES                        PHASE
+cert-manager-operator.v1.13.0   cert-manager Operator for Red Hat OpenShift   1.13.0    cert-manager-operator.v1.12.1   Succeeded
+
+
+oc get pods -n cert-manager-operator
+NAME                                                        READY   STATUS    RESTARTS   AGE
+cert-manager-operator-controller-manager-695b4d46cb-r4hld   2/2     Running   0          7m4s
+
+
+oc get pods -n cert-manager
+NAME                                       READY   STATUS    RESTARTS   AGE
+cert-manager-58b7f649c4-dp6l4              1/1     Running   0          7m1s
+cert-manager-cainjector-5565b8f897-gx25h   1/1     Running   0          7m37s
+cert-manager-webhook-9bc98cbdd-f972x       1/1     Running   0          7m40s
```

Comment:
do you think it would help if these verification commands are broken down to highlight what to look out for while verifying?
e.g. in `oc get pods -n cert-manager` IIUC, we need to check for the presence of all 3 pods as well as the status `running`.

Reply:
yeah, that would be a plus to have. But I guess the sample output will be self-explanatory to compare with the actual output that the readers will have. 

There will be 4-5 more sentences if I'll break it :(